tests/record_test.py

```diff
@@ -17,13 +17,13 @@ def test_create_xml_string(self):
         field = PYUNTL_DISPATCH['title'](content=None)
         self.record.add_child(field)
         xml = self.record.create_xml_string()
-        self.assertTrue(xml.strip() == ('<?xml version="1.0" encoding='
-                                        '"UTF-8"?>\n<metadata>\n'
-                                        '  <title/>\n</metadata>\n').strip())
+        self.assertTrue(xml.strip(), (b'<?xml version="1.0" encoding='
```

Comment:
So I don't think this is right. We either need to go back to using the `==` so we're passing an expression which will then get evaluated to True or False, or we need to use `self.assertEqual` if we want to pass in the two arguments as they are.